README.md

```diff
@@ -13,62 +13,126 @@
         <img src="https://img.shields.io/badge/Contributions-welcome-blueviolet" /></a>
 </p>
 
-# QuickBooks ([docs](https://dbt-quickbooks.netlify.app/))
-
-This package models QuickBooks data from [Fivetran's connector](https://fivetran.com/docs/applications/quickbooks). It uses data in the format described by [this ERD](https://fivetran.com/docs/applications/quickbooks#schemainformation).
-
-The main focus of this package is to provide users insights into their QuickBooks data that can be used for financial statement reporting and deeper analysis. The package achieves this by:
+# QuickBooks dbt Package ([Docs](https://fivetran.github.io/dbt_quickbooks/))
```

Comment:
I just realized the `Fivetran Release Phase: Beta` is included in the badges. This may be a hold over from a prior release, but we no longer want to use these badges. Let's remove that line above.

```html
<a alt="Fivetran-Release"
        href="https://fivetran.com/docs/getting-started/core-concepts#releasephases">
        <img src="https://img.shields.io/badge/Fivetran Release Phase-_Beta-orange.svg" /></a>
```

Reply:
Removed!